thoth/investigator/common.py

```diff
@@ -180,7 +181,7 @@ async def _schedule_revsolver(openshift: OpenShift, package_name: str, package_v
         _LOGGER.exception(
             "Failed to schedule reverse solver for %r in version %r: %r", package_name, package_version, e
         )
-        is_scheduled = 0
+        raise e
```

Comment:
can we keep setting 0 to is_scheduled metric in case of exceptions? We will have holes in the metrics otherwise. WDYT?

Reply:
I assume we want the exception re-raised. Is there a good time to do this?

Reply:
This will be handled in a later PR